src/test/java/com/iota/iri/service/snapshot/impl/SnapshotMockUtils.java

```diff
@@ -0,0 +1,64 @@
+package com.iota.iri.service.snapshot.impl;
+
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.model.Hash;
+import com.iota.iri.service.snapshot.Snapshot;
+import com.iota.iri.service.snapshot.SnapshotProvider;
+import org.mockito.Mockito;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SnapshotMockUtils {
+    private static final int DEFAULT_MILESTONE_START_INDEX = 70000;
+
+    private static final Hash DEFAULT_GENESIS_HASH = Hash.NULL_HASH;
+
+    private static final long DEFAULT_GENESIS_TIMESTAMP = 1522146728;
+
+    //region [mockSnapshotProvider] ////////////////////////////////////////////////////////////////////////////////////
+
+    public static SnapshotProvider mockSnapshotProvider(SnapshotProvider snapshotProvider) {
+        return mockSnapshotProvider(snapshotProvider, DEFAULT_MILESTONE_START_INDEX);
+    }
+
+    public static SnapshotProvider mockSnapshotProvider(SnapshotProvider snapshotProvider, int milestoneStartIndex) {
+        return mockSnapshotProvider(snapshotProvider, milestoneStartIndex, DEFAULT_GENESIS_HASH);
+    }
+
+    public static SnapshotProvider mockSnapshotProvider(SnapshotProvider snapshotProvider, int milestoneStartIndex,
+            Hash genesisHash) {
+
+        return mockSnapshotProvider(snapshotProvider, milestoneStartIndex, genesisHash, DEFAULT_GENESIS_TIMESTAMP);
+    }
+
+    public static SnapshotProvider mockSnapshotProvider(SnapshotProvider snapshotProvider, int milestoneStartIndex,
+            Hash genesisHash, long genesisTimestamp) {
+
+        Map<Hash, Long> balances = new HashMap<>();
+        balances.put(Hash.NULL_HASH, TransactionViewModel.SUPPLY);
```

Comment:
here you are using it directly

Reply:
Yes because it wouldn't make much sense to introduce a variable for an "address" - there can be a lot of addresses and they dont have a special meaning - there is only one genesis hash however and it actually has a special meaning in the processing of IRI

Reply:
I will add a variable for this default genesis address so opening the SnapshotMockUtils will immediately reveal the important values for the mocked snapshots